inc/interactive/class-content.php

```diff
@@ -73,7 +83,7 @@ static public function hooks( Content $obj ) {
 		// @see https://codex.wordpress.org/Embeds/
 		add_action( 'init', [ $obj, 'registerEmbedHandlers' ] );
 		add_filter( 'oembed_providers', [ $obj, 'addExtraOembedProviders' ] );
-		add_filter( 'oembed_result', [ $obj, 'adjustOembeds' ], 10, 3 );
+		add_filter( 'embed_oembed_html', [ $obj, 'adjustOembeds' ], 10, 3 );
```

Comment:
From: https://wordpress.stackexchange.com/questions/353313/combine-embed-oembed-html-and-oembed-result
- embed_oembed_html filters the cached oEmbed HTML.
- oembed_result filters the HTML returned by the oEmbed provider.
So oembed_result is when the html is first retrieved, before caching. embed_oembed_html is when the cache has been fetched but before rendering.